step.yml

```diff
@@ -1,39 +1,50 @@
 title: Xcode Build for testing for iOS
 summary: Performs xcodebuild's build-for-testing action
 description: |-
-  The Step runs Xcode's `xcodebuild` command with the `build-for-testing` option. This builds your app for testing and also creates an `.xctestrun` file.
+  The Step runs Xcode's `xcodebuild` command with the `build-for-testing` option. This builds your app so that you can, for example, upload it to a third-party testing service to run your app on a real device. 
+  The Step also creates an `.xctestrun` file.
+  To be able to run your app on a real device it needs code signing. The Step logs you into your Apple Developer account based on the [Apple service connection you provide on Bitrise](https://devcenter.bitrise.io/en/accounts/connecting-to-services/apple-services-connection.html) and downloads any provisioning profiles needed for your project based on the **Distribution method**.
```

Comment:
But the default input value for this is "off", shouldn't we highlight it here?

Reply:
To be able to run your app on a real device -> To be able to run your tests on a real device